lib/archiver/archiver.rb

```diff
@@ -29,6 +29,19 @@ def self.allowed_hosts
     @allowed_hosts || []
   end
 
+  def self.public_hosts=(hosts)
+    hosts = [] if hosts.nil?
+    hosts = hosts.split(' ') if hosts.is_a?(String)
+    unless hosts.is_a?(Enumerable) && hosts.all? {|host| host.is_a?(String)}
+      raise StandardError, 'Public hosts must be a string or enumerable of strings'
+    end
```

Comment:
If you keep this (see comment on the `.env.example` file), I think this list parsing logic should be extracted so we don’t duplicate what’s already in the `allowed_hosts` setter.

Reply:
dispensed with this